features/000-anonymous-user/sdk.md

```diff
@@ -0,0 +1,27 @@
+# SDK
+
+## Authentication
+
+APIs would be added to perform authentication:
+```ts
+function authenticateAnonymously(): Promise<{ user: User }>;
```

Comment:
If I understand correctly, we will use browser-less flow for signup and with browser flow for promote. So this function should be step of browser-less flow signup, right?

Would like to know will this function behaves differently when the client app is first party or third party app? `response_type=none` for first party and `response_type=code` for third party app?

Reply:
> So this function should be step of browser-less flow signup, right?

Yes, the SDK only uses browser-less flow.

> Would like to know will this function behaves differently when the client app is first party or third party app? response_type=none for first party and response_type=code for third party app?

Yes, the behavior is same as normal authorization.

Reply:
After the simplification, native application SDK will use the browser-less flow for both signup and login and web application SDK will use the authorization code flow, am I correct?

Reply:
Yes, native application would use browser-less flow. However, for web platform, it is not yet fully specified as I don't have a concrete use-case to demonstrate how would users like the flow to be.